backend/clubs/views.py

```diff
@@ -2707,6 +2724,11 @@ def create_tickets(self, request, *args, **kwargs):
             event.ticket_order_limit = order_limit
             event.save()
 
+        delay_drop = request.data.get("delay_drop", None)
```

Comment:
Why + 12h here? Seems arbitrary. Can we maybe give the user better control on when exactly their tickets drop?

Reply:
No particular reason behind choosing 12. Will modify `delay_drop` to accept an integer value, and use that instead.

Reply:
Why not just a timestamp?

Reply:
Like a Unix timestamp? Sure.

Reply:
Not necessarily a Unix timestamp...there's some use cases around the codebase where we pass in datetime from frontend to backend. For example, we allow club admins to control the `application_start_time`, `application_end_time`, etc. You could use those as guiding examples perhaps